cni/network/network_windows.go

```diff
@@ -192,3 +204,17 @@ func getPoliciesFromRuntimeCfg(nwCfg *cni.NetworkConfig) []policy.Policy {
 
 	return policies
 }
+
+func getCustomDNS(nwCfg *cni.NetworkConfig) network.DNSInfo {
+	log.Printf("[net] RuntimeConfigs: %+v", nwCfg.RuntimeConfig)
+
+	var search string
+	if len(nwCfg.RuntimeConfig.DNS.Searches) > 0 {
+		search = nwCfg.RuntimeConfig.DNS.Searches[0]
+	}
```

Comment:
What about the rest elements in Searches?
We concatenates the DNS.Search in [cni/network/network_windows.go](https://github.com/Azure/azure-container-networking/blob/bfb3eaafa7abdc2ef976e56b02e83cbabf4bce52/cni/network/network_windows.go#L157).
```
	if len(nwCfg.DNS.Search) > 0 {
		epDNS = network.DNSInfo{
			Servers: nwCfg.DNS.Nameservers,
			Suffix:  namespace + "." + strings.Join(nwCfg.DNS.Search, ","),
		}
	}
```

Reply:
I noticed that in `getNetworkDNSSettings` [servers is set and suffixes isn't](https://github.com/Azure/azure-container-networking/blob/7640ce9b955c57c74261cb74dee4a69478a60086/cni/network/network_windows.go#L149). Should we be doing the same thing? If we use our current code, then `Suffix` could possibly be set in `getNetworkDNSSettings`.

Also, for `getEndpointDNSSettings` [domain is the value for suffix](https://github.com/Azure/azure-container-networking/blob/7640ce9b955c57c74261cb74dee4a69478a60086/cni/network/network_windows.go#L176), but we do not have a domain for our custom DNS.

Does our current logic in `getCustomDNS` make sense if we concatenate the searches? Our end result would look like this:
```go
func getCustomDNS(nwCfg *cni.NetworkConfig) network.DNSInfo {
	log.Printf("[net] RuntimeConfigs: %+v", nwCfg.RuntimeConfig)

	var search string
	if len(nwCfg.RuntimeConfig.DNS.Searches) > 0 {
		search = strings.Join(nwCfg.RuntimeConfig.DNS.Searches, ",")
	}

	return network.DNSInfo{
		Servers: nwCfg.RuntimeConfig.DNS.Servers,
		Suffix:  search,
	}
}
```

Reply:
Previously, HNSNetwork's DNS suffix is overwritten by HNSEndpoint's DNS suffix. That's why we don't set it in getNetworkDNSSettings. I don't know if this is still true. If it does, then we don't need to set suffix for network.